docs/develop/dotnet/schedules.mdx

```diff
@@ -64,6 +64,13 @@ var handle = await client.CreateScheduleAsync(
         }));
 ```
 
+:::tip Schedule Auto-Deletion
```

Comment:
maybe make this a note. I'm looking at the preview and generally it seems like we've been using tips for statuses.

Reply:
I don't think this is important enough for a warning as it is an implementation detail. It's a hint to the user not to rely on the schedule existing after no more firing times will happen.

I'm going to leave it as is based on the following criteria.

**Notes** (spelled `:::note`) add supplemental information that is not cautionary. They are the least important of the call out styles and reserved for use only when another, more fitting, style can’t be found.

**Tips** (spelled `:::tip`) play the role of hints to the user, to enhance the efficiency of their work with optional improvements.

Reply:
Ok going to unblock this! We should come to a consensus on the admonitions to avoid misuse by contributors.

Reply:
@jsundai Reference: https://www.notion.so/temporalio/Temporal-Documentation-Markdown-Guide-2a5eee3d1ddf4e0d8827fd1b2b34a3f4?pvs=4#01226b60121d45f9bc75c6e93e1f0826